_posts/2022-05-12-data-pipelines-in-Databricks.markdown

```diff
@@ -0,0 +1,240 @@
+---
+layout: post
+title: "Data pipelines in Databricks"
+date: 2022-05-12
+tags: Databricks data testing
+author: Natalie Laing
+image: /img/code.jpg
+---
+
+# Data pipelines in Databricks
+
+## Introduction
+
+![delta lake](/img/blog/databricks-data-pipeline/delta-lake.png)
```

Comment:
I wonder if adding a more Databricks specific picture would be better here since it is the introduction. You could use this image where you are explaining the differences between bronze, silver and gold